src/Microsoft.ComponentDetection.Detectors/cocoapods/PodComponentDetector.cs

```diff
@@ -404,7 +404,9 @@ public string GetSpecRepositoryOfSpec(string specName)
                     {
                         // CocoaPods specs are stored in a git repo but depending on settings/CocoaPods version
                         // the repo is shown differently in the Podfile.lock
+#pragma warning disable CA1308
                         return repository.Key.ToLowerInvariant() switch
```

Comment:
I think you could use `ToUpperInvariant()` here as well, and change the switch case from `trunk` to `TRUNK`.